classes/label.rb

```diff
@@ -0,0 +1,17 @@
+require_relative 'item'
+
+class Label
+  attr_reader :title, :color, :id
+  attr_accessor :items
+
+  def initialize(title, color)
+    @id = rand(1000)
+    @title = title
+    @color = color
+    @items = []
+  end
+
+  def add_item(item)
+    item.add_label(self)
+  end
```

Comment:
- [ ] to set up the 1-to-many relationships for `Label` you need to push `item` to `@items` as well as you did in the `Genre` class as well.